kingpin/actors/aws/elb.py

```diff
@@ -66,6 +66,78 @@ class ELBBaseActor(base.AWSBaseActor):
         'region': (str, REQUIRED, 'AWS region (or zone) name, like us-west-2')
     }
 
+    @gen.coroutine
+    def _check_access(self, elb):
+        """Perform a dummy operation to check credential accesss.
+
+        Intended to be used in a dry run, this method attempts to perform an
+        invalid set_listener call and monitors the output of the error. If the
+        error is anything other than AccessDenied then the provided credentials
+        are sufficient and we do nothing.
+
+        Args:
+            elb: boto LoadBalancer object.
+        """
+        try:
+            # A blank ARN value should have code 'CertificateNotFound'
+            # We're only checking if credentials have sufficient access
+            yield self.thread(
+                elb.set_listener_SSL_certificate,
+                self.option('port'),
```

Comment:
The problem with this is that if you use the `_check_access()` method in any other AWS actor, it may not have the `port` option. Perhaps just fake the port out with something invalid?